packages/server/knexfile.js

```diff
@@ -26,6 +26,13 @@ function walk( dir ) {
 
 let migrationDirs = walk( './modules' )
 
+let env = process.env
```

Comment:
As above, no need to add this code you can use the `POSTGRES_URL`

Reply:
Yeah, I think I put down a note somewhere that I don't like the name of the env variable `POSTGRES_URL` while it is actually more of a connection string.